client/src/app/site/motions/models/view-motion.ts

```diff
@@ -340,6 +346,34 @@ export class ViewMotion extends BaseViewModelWithAgendaItemAndListOfSpeakers<Mot
         this._diffLines = value;
     }
 
+    /**
+     * Determine if a motion has a parent at all
+     */
+    public get hasParent(): boolean {
+        return !!this.parent_id;
+    }
+
+    /**
+     * Determine if a motion has amenments
+     */
+    public get hasAmendments(): boolean {
+        return !!this.amendments && !!this.amendments.length;
+    }
+
+    /**
+     * Determine if the motion has parents, is a parent of neither
+     */
+    public get motionType(): number {
+        if (this.hasAmendments) {
+            return MotionType.Parent;
+        } else if (this.hasParent) {
+            return MotionType.Amendment;
+        } else {
+            // not any amendment
+            return MotionType.Normal;
```

Comment:
I think you can remove the "normal" type.

Reply:
ok